scripts/generate_mappings.py

```diff
@@ -0,0 +1,97 @@
+import json
+from collections import defaultdict
+from ontology_mapper.loader import NugetPackage
+from ontology_mapper.engine import MappingEngine
+
+with NugetPackage('mapped.ontologies.core.dtdl') as package:
+    mapped_ontology = package.get_content('1.23.8')
+
+with open('data/mapped_v1_dtdlv2_Willow.json') as file:
+    mapped_mappings = json.load(file)
+    mapped_manual_mappings = []
+    for mapping in mapped_mappings['InterfaceRemaps']:
+        if mapping.get('IsInferred', False) == False:
+            mapping['IsInferred'] = False
+            mapped_manual_mappings.append(mapping)
+
+with NugetPackage('willowinc.ontology.dtdlv3') as package:
```

Comment:
Specify the version

Reply:
@jbkoh This is the nuget package name (https://www.nuget.org/packages/WillowInc.Ontology.DTDLv3). If the `get_content()` method does not specify the version, it will download the most recent version.